### PR TITLE
Implement FileStore

### PR DESCRIPTION
**Pull request by mgatny**
- [x] Incr/Set MsgSeqNums
- [x] Refresh
- [x] Reset
- [x] SaveMessage
- [x] GetMessages
- [x] Move some of the file-manipulation funcs into `fileutil.go`
- [x] Update `config/doc.go` for `FileStorePath`

Refs #32

### CODE REVIEW

**Comment by mgatny** [comment]
:eyes: please, @cbusbey

File: config/configuration.go [hunk 1]
```diff
@@ -15,4 +15,5 @@ const (
 	ResetOnLogon            string = "ResetOnLogon"
 	HeartBtInt              string = "HeartBtInt"
 	FileLogPath             string = "FileLogPath"
+	FileStorePath           string = "FileStorePath"
```
Comment (cbusbey) [inline config/configuration.go]: Can you add this to config/doc.go?  Relying on godoc for all documentation.

Comment (mgatny) [inline config/configuration.go]: :+1: done:
https://github.com/mgatny/quickfixgo-quickfix/blob/filestore/config/doc.go#L122-L124

**Comment by cbusbey** [comment]
:horse: :meat_on_bone:

**Pull request merged**